packages/contentstack-auth/src/commands/auth/logout.ts

```diff
@@ -60,9 +60,22 @@ export default class LogoutCommand extends Command {
         }
       }
     } catch (error) {
-      logger.error('Logout failed', error.message);
+      let errorMessage = '';
+      if (error) {
+        if (error.message) {
+          if (error.message.message) {
+            errorMessage = error.message.message;
+          } else {
+            errorMessage = error.message;
+          }
+        } else {
+          errorMessage = error;
+        }
+      }
```

Comment:
Suggestion: This particular part of the code is repeating in both login in and logout, It can be reused [**Minor**]

https://github.com/contentstack/cli/pull/725/files#diff-a01e7a0777da625b321b5da6d541e83c12ffcb2284a5f0a556c911beb449d328R67-R78

Reply:
Sure, we will change this in next commits of error handling